client/app/scripts/sir-trevor-blocks/index.js

```diff
@@ -74,7 +74,7 @@ function isURI(string) {
 }
 
 function replaceEmbedWithUrl(string) {
-    var generalPattern = new RegExp(socialEmbedRegex, 'i');
+    var generalPattern = new RegExp(socialEmbedRegex, 'si');
```

Comment:
what is `s` from? ....

Reply:
s stands for new lines. It is not necessary bcs there are new lines included in regex but just in case...